server/src/main/java/io/druid/server/listener/resource/ListenerHandler.java

```diff
@@ -35,5 +35,9 @@
   Response handleGET(String id);
   Response handleGETAll();
   Response handleDELETE(String id);
+  Response handleUpdates(InputStream inputStream, ObjectMapper mapper);
```

Comment:
yeah, I'm not sure if this framework can fit that with purity. but then this framework is only utilized for lookup listener and I'm not sure if we just should not use this framework. IMHO that framework does make things a little bit convoluted and my understanding is that it was created with the hope of reuse in something.